packages/terra-table/tests/wdio/table-spec.js

```diff
@@ -104,6 +104,28 @@ Terra.describeViewports('Table', ['medium', 'large'], () => {
     });
   });
 
+  describe('Focusable Cell Element Navigation : ', () => {
+    const focusElementSelector = 'body';
+
+    beforeEach(() => {
+      browser.url('/raw/tests/cerner-terra-framework-docs/table/table-focusable-cell');
+    });
+
+    it('validates that the proper grid element is selected when Tab is used to give initial focus to the grid', () => {
+      browser.keys(['Tab']);
+
+      expect(browser.$$('button:focus')).toBeElementsArrayOfSize(1);
+      Terra.validates.element('table-focus-first-button', { columnResizeSelector: focusElementSelector });
+    });
+
+    it('validates that the proper grid element is selected when Tab is used to give initial focus to the grid', () => {
```

Comment:
Can we update this to differentiate it from [line 114](https://github.com/cerner/terra-framework/pull/1850/files#diff-b5d6f33478c38faf7a2adf314d4c30af172aa9b9f80d757af5decb595428c64eR114)?